models/ed/R/met2model.ED2.R

```diff
@@ -27,13 +27,9 @@
 ##' @param end_date the end date of the data to be downloaded (will only use the year part of the date)
 ##' @param overwrite should existing files be overwritten
 ##' @param verbose should the function be very verbose
-met2model.ED2 <- function(in.path,in.prefix,outfolder,start_date, end_date, lst=0,overwrite=FALSE,verbose=FALSE){
+met2model.ED2 <- function(in.path,in.prefix,outfolder,start_date, end_date, lst="lst=0",..., overwrite=FALSE,verbose=FALSE){
```

Comment:
I don't understand the change from lst=0 to lst="lst=0". What's the advantage of doing this, and then having to do the more complex as.numeric(eval(parse(text=lst))? It seems like they do the same thing but the new way is more complicated and less intuitive.